rosa-workshop/rosa/12-delete_cluster.md

```diff
@@ -8,7 +8,7 @@ To delete a ROSA cluster follow the steps below.
 
 2. Once you know which one to delete run
 
-	`rosa delete cluster <clustername>`
+	`rosa delete cluster --cluster-name=<cluster name>`
```

Comment:
This should be "--cluster=" instead of "--cluster-name=".

Reply:
You're right. My mistake. I'll close this.